src/components/SettingsPage.vue

```diff
@@ -237,24 +222,478 @@
+                    aria-controls="panel-1"
+                  >
+                    <span class="icon-title h6 mb-0 font-weight-bold"
+                      >Update Relationship Status</span
+                    >
+                  </a>
+                  <div class="collapse" id="panel-15">
+                    <div class="pt-3">
+                      <fieldset>
+                        <legend class="h6">Choose your Status</legend>
+                        <div class="form-check">
+                          <input
+                            class="form-check-input"
+                            type="radio"
+                            v-model="relationshipStatus_single"
+                            name="exampleRadios1"
+                            id="exampleRadios4"
+                            value="single"
+                            checked=""
+                          />
+                          <label class="form-check-label" for="exampleRadios4">
+                            Single
+                          </label>
+                        </div>
+                        <div class="form-check">
+                          <input
+                            class="form-check-input"
+                            type="radio"
+                            v-model="relationshipStatus_engaged"
+                            name="exampleRadios1"
+                            id="exampleRadios5"
+                            value="Engaged"
+                          />
+                          <label class="form-check-label" for="exampleRadios5">
+                            Engaged
+                          </label>
+                        </div>
+                        <div class="form-check">
+                          <input
+                            class="form-check-input"
+                            type="radio"
+                            v-model="relationshipStatus_married"
+                            name="exampleRadios1"
+                            id="exampleRadios6"
+                            value="married"
+                          />
+                          <label class="form-check-label" for="exampleRadios6">
+                            Married
+                          </label>
+                        </div>
+                        <div class="form-check">
+                          <input
+                            class="form-check-input"
+                            type="radio"
+                            v-model="relationshipStatus_divorced"
+                            name="exampleRadios1"
+                            id="exampleRadios7"
+                            value="divorced"
+                          />
+                          <label class="form-check-label" for="exampleRadios7">
+                            Divorced
+                          </label>
+                        </div>
+                        <div class="form-check">
+                          <input
```

Comment:
@itisdb / @ayushi-siddhu why are choice field form elements treated as inputs and not as options ? Please don't use any random element to fit into template which turns out to be actual useless at future while integration. Extra time is involved now to repair it and it becomes dependency to some other developers as well !